app/server/tests/api-deletePost.test.js

```diff
@@ -0,0 +1,69 @@
+const { AuthFunctions } = require('./functions/authFunctions');
+const { password, name, email } = require('./functions/randomizer');
+const { PostFunctions } = require('./functions/postFunctions');
+const community = require('./functions/communityFunctions');
+
+let auth = new AuthFunctions();
+let post = new PostFunctions();
+
+describe('Testing user\'s ability to delete posts from their communities', () => {
+    let loginResponse;
+    let response;
+    let userName = name.gen();
+    let userPassword = password.gen();
+    let postResponse;
+
+    beforeAll(async () => {
+        // Set up user
+        await auth.register(userName, email.gen(), userPassword);
+        loginResponse = await auth.login(userName, userPassword);
+        // Create some communities
+        await community.createCommunity('Thundercats', 'Meow', 'Always feeding time', [], loginResponse.body.token);
```

Comment:
love the theming, just remember this repository is 100% public. anyone can see this.

Reply:
So is this inappropriate?

Reply:
Not saying inappropriate, but you may be toeing the line. Let me ask above.

Reply:
Easy thing to change. Trying to keep it fresh. :)